### PR TITLE
Updated some NuGet packages

### PR DESCRIPTION
**Pull request by ErikMogensen**
* Updated some NuGet packages including WindowsAzure.ServiceBus

* Removed Microsoft.WindowsAzure.ConfigurationManager

* Fixed the warning about App.Config when building

* GitVersionTask is deprecated even though I updated it to the latest version.

### CODE REVIEW

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 5.1.0-UpdatedSomeNuGetPackages.13.build.769 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/37132865) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/b4a11bc5cd by @ErikMogensen)

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 5.1.0-UpdatedSomeNuGetPackages.15.build.771 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/37133063) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/821aaf54a8 by @ErikMogensen)

**Comment by SeanFeldman** [comment]
> GitVersionTask is deprecated even though I updated it to the latest version.

I think what's happening is GitVersionTask is transformed into GitVersion.MsBuild. You'll notice it's a [new package](https://www.nuget.org/packages/GitVersion.MsBuild/) but the version is continued. Both packages are providing the functionality for [GitVersion](https://github.com/GitTools/GitVersion/releases) which is live and well.

File: src/Common/Common.csproj [hunk 1]
```diff
@@ -74,17 +74,17 @@
...
-      <HintPath>..\packages\WindowsAzure.ServiceBus.6.0.0\lib\net462\Microsoft.ServiceBus.dll</HintPath>
+      <HintPath>..\packages\WindowsAzure.ServiceBus.6.0.3\lib\net462\Microsoft.ServiceBus.dll</HintPath>
     </Reference>
     <Reference Include="Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\packages\Newtonsoft.Json.12.0.2\lib\net45\Newtonsoft.Json.dll</HintPath>
+      <HintPath>..\packages\Newtonsoft.Json.12.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.Buffers, Version=4.0.3.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
-      <HintPath>..\packages\System.Buffers.4.5.0\lib\netstandard2.0\System.Buffers.dll</HintPath>
```
Comment (SeanFeldman) [inline src/Common/Common.csproj:87]: Was it intentionally changed from netstandard to net461?

File: src/Common/Common.csproj [hunk 2]
```diff
@@ -99,7 +99,7 @@
       <HintPath>..\packages\System.IdentityModel.Tokens.Jwt.5.4.0\lib\net461\System.IdentityModel.Tokens.Jwt.dll</HintPath>
     </Reference>
     <Reference Include="System.Memory, Version=4.0.1.1, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
-      <HintPath>..\packages\System.Memory.4.5.3\lib\netstandard2.0\System.Memory.dll</HintPath>
```
Comment (SeanFeldman) [inline src/Common/Common.csproj:102]: same question here

File: src/Common/Common.csproj [hunk 3]
```diff
@@ -168,8 +168,8 @@
     <Reference Include="System.Threading.Channels, Version=4.0.1.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
       <HintPath>..\packages\System.Threading.Channels.4.6.0\lib\netstandard2.0\System.Threading.Channels.dll</HintPath>
     </Reference>
-    <Reference Include="System.Threading.Tasks.Extensions, Version=4.2.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
-      <HintPath>..\packages\System.Threading.Tasks.Extensions.4.5.2\lib\netstandard2.0\System.Threading.Tasks.Extensions.dll</HintPath>
```
Comment (SeanFeldman) [inline src/Common/Common.csproj:172]: and here

**Review by SeanFeldman — APPROVED** [review]

File: src/Common/Common.csproj (hunk 1 above)
Comment (ErikMogensen) [reply src/Common/Common.csproj:87]: No, it did that automatically when I selected to update it.

File: src/Common/Common.csproj (hunk 2 above)
Comment (ErikMogensen) [reply src/Common/Common.csproj:102]: Same answer here, it did that automatically when I selected to update it.

File: src/Common/Common.csproj (hunk 3 above)
Comment (ErikMogensen) [reply src/Common/Common.csproj:172]: Same answer here, it did that automatically when I selected to update it.

**Review by paolosalvatori — APPROVED** [review]

**Pull request merged**